test/setup_travis.sh

```diff
@@ -44,7 +50,7 @@ function setup_docker {
 
 case $TEST in
   *chrome*)
-    setup_chrome
+    setup_chrome $TEST
```

Comment:
```suggestion
    setup_chrome "$TEST"
```

Reply:
Thanks. saw it again this morning realizing i missed quotes